sass/main.scss

```diff
@@ -19,18 +19,82 @@
 	}
```

Comment:
this file has a lot of extra returns.  be consistent.  no empty line before closing curly.  single line between each classs